docs/smart-contracts/v12/OutpostL2.md

```diff
@@ -0,0 +1,291 @@
+# Outpost L2 contracts
+[TOC]
+
+!!! danger "Warning"
+    The code is not audited yet and it is not meant to use in production. Current status: release-candidate
+
+## Motivation
+This document outlines the smart contract changes introduced in the `feature/outposts` branch. These changes introduce new administrative functions for managing the Local Exit Tree (LET), claims, and local balance trees. The aggkit must **listen to the events** emitted by these functions to keep its database synchronized with the contract state.
```

Comment:
branch `feature/outposts`should not be mentioned. Otherwise, we will keep changing this doc.

Reply:
Link to the release at the top of the document should be fine.
I also miss links to the SC, events, params in the whole document.

Given the release tag at the top of the document, it will be easy to replace quickly all over the document